src/jit/lsraxarch.cpp

```diff
@@ -2263,9 +2290,8 @@ void LinearScan::BuildSIMD(GenTreeSIMD* simdTree)
 // Return Value:
 //    None.
 
-void LinearScan::BuildHWIntrinsic(GenTreeHWIntrinsic* intrinsicTree)
+int LinearScan::BuildHWIntrinsic(GenTreeHWIntrinsic* intrinsicTree)
```

Comment:
I cleaned up this method here: https://github.com/dotnet/coreclr/pull/18078

It might be worthwhile merging the PRs together (some of the changes were fixing "correctness" issues I found, and the other changes are required to support the FMA instruction set, where any of the three operands can be contained).

Reply:
The changes I did to support FMA are here: https://github.com/tannergooding/coreclr/commit/e402548087f2bf9554e83052911561c016c60346#diff-498fd4859cb147c9bf082f5ebf32ca8fR2507

(The PR isn't up as it requires the CoreFX change adding back the unimplemented APIs to the ref assembly).

Reply:
> It might be worthwhile merging the PRs together

I think it might be easier to serialize them.